axelor-message/src/main/java/com/axelor/apps/message/service/TemplateMessageServiceImpl.java

```diff
@@ -182,7 +182,7 @@ public Message generateMessage(Long objectId, String model, String tag, Template
     Message message =
         messageService.createMessage(
             model,
-            Long.valueOf(objectId).intValue(),
+            objectId.intValue(),
```

Comment:
Below you replaced the same expression with `Math.toIntExact(objectId)`, which expression is the best ?

Reply:
In this case, `objectId` is a `Long` whereas in `GenerateMessageController` it is a `long`. Since `Math.toIntExact(objectId)` can be used in both case and is safer, we should use it in both cases.

https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#toIntExact-long-

Reply:
Can you do it please ? :)

Reply:
Yes I can :)